packages/appstore/src/components/main-title-bar/asset-summary.tsx

```diff
@@ -27,7 +28,7 @@ const AssetSummary = observer(() => {
     const cr_account = !is_eu_user && !no_CR_account;
 
     //dont show loader if user has no respective regional account
-    if ((is_switching || is_logging_in) && (eu_account || cr_account)) {
+    if (((is_switching || is_logging_in) && (eu_account || cr_account)) || !is_landing_company_loaded || !last_update) {
```

Comment:
@shontzu-deriv We have a loader for assetSummary when we switch between real and demo, need to add these conditions to that part instead of loader for the whole page
<img width="1171" alt="image" src="https://github.com/binary-com/deriv-app/assets/82078941/0899ec9e-cc89-4107-8f4e-f836dbff7e3f">

Reply:
it is in the asset summary component already

Reply:
@shontzu-deriv my bad, you already added it in the correct place 👍